src/App.js

```diff
@@ -1,12 +1,15 @@
 import React, { useState } from "react";
 import "./styles.css";
 import { Modal } from "./modal/modal";
-import {days as dayKeys, fullDays, defaultTasks} from './helper.js';
+import {days as dayKeys, fullDays} from './helper.js';
 import TaskCard from "./task-card.component";
+import useGetTasks from "./use-get-tasks";
```

Comment:
react custom hook that fetches tasks from restdb

Reply:
I'm not very good at custom hooks, something I need to study more.